packages/site/src/layout/SearchBox.tsx

```diff
@@ -24,13 +21,20 @@ import { hooksList } from "../hooksList";
 import { packagesList } from "../packagesList";
 import { ContentListItem } from "../types/components";
 
+const lists = [
```

Comment:
`lists` is static; keeping it outside the function helps avoids unnecessary re-evaluation on every render